test/server/event-stream.test.ts

```diff
@@ -36,12 +37,21 @@ describe(eventStream, () => {
 		if (!response.body) throw new Error("Response body is undefined");
 
 		let reader = response.body.getReader();
+		let decoder = new TextDecoder();
 
 		let { value: event } = await reader.read();
-		expect(event).toEqual(new TextEncoder().encode("event: message\n"));
+		expect(decoder.decode(event)).toEqual("event: message\n");
```

Comment:
Using a decoder on the actual instead of using an encoder on the expected gives human-readable diffs of strings. It's very hard to understand what's wrong by looking at the Int8 array diff.